allennlp/models/ensemble/bidaf_ensemble.py

```diff
@@ -0,0 +1,111 @@
+from overrides import overrides
+import torch
+from typing import Dict, List, Any
+
+from allennlp.models.ensemble.ensemble import Ensemble
+from allennlp.models.archival import load_archive
+from allennlp.models import Model
+from allennlp.models.reading_comprehension import BidirectionalAttentionFlow
+from allennlp.common import Params
+from allennlp.data import Vocabulary
+from allennlp.training.metrics import SquadEmAndF1
+
+@Model.register("bidaf-ensemble")
+class BidafEnsemble(Ensemble):
+
+    def __init__(self,
+                 submodels: List[BidirectionalAttentionFlow]) -> None:
+        super(BidafEnsemble, self).__init__(submodels)
+
+        # Setting this propagates calls to .eval() so dropout is disabled on the submodels in evaluation
+        # and prediction.
+        self.linears = torch.nn.ModuleList(submodels)
+
+        self.submodels = submodels
```

Comment:
You should be able to just do `self.submodels = torch.nn.ModuleList(submodels)`, without having `self.linears` at all.